.changelog/196.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+linux: fix linux capabilities
```

Comment:
As a reader of the release notes, I think I would prefer to have a few more details about what was broken.